specification/_global/bulk/types.ts

```diff
@@ -56,3 +63,27 @@ export class BulkCreateResponseItem extends BulkResponseItemBase {}
 export class BulkUpdateResponseItem extends BulkResponseItemBase {}
 
 export class BulkDeleteResponseItem extends BulkResponseItemBase {}
+
+export class BulkOperation {
+  _id: Id
+  _index: IndexName
+  retry_on_conflict: integer
+  routing: Routing
+  version: VersionNumber
+  version_type: VersionType
+}
+
+export class BulkOperationContainer {
```

Comment:
don't we need to annotate this with `/** @variants container */` ?

Reply:
Great catch!